src/pages/loadFactory/index.tsx

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { CheckCircleIcon } from '@patternfly/react-icons';
+import { ExclamationCircleIcon, InProgressIcon } from '@patternfly/react-icons/dist/js/icons';
+import React, { RefObject } from 'react';
+import {
+  PageSection,
+  PageSectionVariants,
+  Tabs,
+  Tab,
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+  AlertVariant,
+  Wizard, WizardStep,
+} from '@patternfly/react-core';
+import Header from '../../components/Header';
+import { WorkspaceStatus } from '../../services/workspaceStatus';
+import LogsTab from '../../components/LogsTab';
+
+import styles from '../../components/WorkspaceStatusLabel/WorkspaceStatusLabel.module.css';
+import './index.styl';
+
+export const SECTION_THEME = PageSectionVariants.light;
+
+export enum LoadFactoryTabs {
+  Progress = 0,
+  Logs = 1,
+}
+
+type Props = {
+  hasError: boolean,
+  currentStep: number,
+  workspaceName: string;
+  workspaceId: string;
+  devfileLocationInfo?: string;
+  callbacks?: {
+    showAlert?: (variant: AlertVariant.success | AlertVariant.danger, title: string) => void
+  }
+};
+
+type State = {
+  alertVisible?: boolean;
+  activeTabKey?: LoadFactoryTabs;
+  stepIdReached?: number;
```

Comment:
Could you elaborate a bit on what `stepIdReached` is used for?

Reply:
I will remove it. It is just a copy/paste error.